README.md

```diff
@@ -1,8 +1,125 @@
-This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Auth Link Portal
+
+[![License](https://img.shields.io/github/license/pesu-dev/auth-link-portal)](https://github.com/pesu-dev/auth-link-portal/blob/main/LICENSE)
+[![Contributors](https://img.shields.io/github/contributors/pesu-dev/auth-link-portal)](https://github.com/pesu-dev/auth-link-portal/graphs/contributors)
+[![Issues](https://img.shields.io/github/issues/pesu-dev/auth-link-portal)](https://github.com/pesu-dev/auth-link-portal/issues)
+[![Project Board](https://img.shields.io/badge/project-board-blue)](https://github.com/orgs/pesu-dev/projects/4/views/11)
+
+This is a website that handles verification of the students joining PESU Discord Server, by authenticating them using Discord OAuth thus providing access to the rest of the server. This project is currently, actively maintained and updated.
 
+The bot is built with security and privacy in mind, ensuring safe and effective community management while maintaining user confidentiality.
+
+me
 ## Getting Started
 
-First, run the development server:
+### For Users
+
+This is currently active in the [PESU Discord Server](https://discord.gg/eZ3uFs2). You can click on the first message in `#welcome`
```

Comment:
instead of saying "click on the first message in welcome", say "click on this link" and hyperlink the netlify website directly